src/pineko/fonll.py

```diff
@@ -0,0 +1,290 @@
+"""Module to manage FONLL predictions."""
+
+import copy
+import json
+import logging
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import pineappl
+import rich
+import yaml
+
+from . import configs
+
+logger = logging.getLogger(__name__)
+
+
+class FONLLInfo:
+    """Class containing all the information for FONLL predictions."""
+
+    def __init__(
+        self, ffns3, ffn03, ffns4, ffns4til, ffns4bar, ffn04, ffns5, ffns5til, ffns5bar
+    ) -> None:
+        """Initialize fonll info."""
+        self.ffns3 = ffns3
+        self.ffn03 = ffn03
+        self.ffns4 = ffns4
+        self.ffns4til = ffns4til
+        self.ffns4bar = ffns4bar
+        self.ffn04 = ffn04
+        self.ffns5 = ffns5
+        self.ffns5til = ffns5til
+        self.ffns5bar = ffns5bar
+
+    @property
+    def fk_paths(self):
+        """Returns the list of the fk paths needed to produce FONLL predictions."""
+        paths = {
+            "ffns3": self.ffns3,
+            "ffn03": self.ffn03,
+            "ffns4": self.ffns4,
+            "ffns4til": self.ffns4til,
+            "ffns4bar": self.ffns4bar,
+            "ffn04": self.ffn04,
+            "ffns5": self.ffns5,
+            "ffns5til": self.ffns5til,
+            "ffns5bar": self.ffns5bar,
+        }
+        actually_existing_paths = [p for p in paths if paths[p] is not None]
+        for p in paths:
+            if p not in actually_existing_paths:
+                logger.warning(
+                    f"Warning! FK table for {p} does not exist and thus is being skipped."
+                )
+        return {p: Path(paths[p]) for p in actually_existing_paths}
```

Comment:
I think this warning is a bit much since it's printed each time `fk_paths` is used. 
```suggestion
        return {p: Path(paths[p]) for p in paths if paths[p] is not None}
```

Reply:
Is [790c0a5](https://github.com/NNPDF/pineko/pull/99/commits/790c0a5f6cb5d8b41eb3e9158662c3a865605c3f) better?

Reply:
yes, thanks!